lib/render/vp2ShaderFragments/Float4ToFloatW.xml

```diff
@@ -15,77 +14,34 @@
             <function_name val="Float4ToFloatW"/>
             <source>
                 <![CDATA[
-// Check whether we're in a GLSL context. If so, we don't expect "floatN"
-// types to exist, so we use "vecN" types instead.
-#if defined(__VERSION__) && (__VERSION__ >= 110)
-#define float4 vec4
-#endif
-
-float
-Float4ToFloatW(float4 input)
+float Float4ToFloatW(float4 input)
 {
     return input.w;
 }
-
 ]]>
             </source>
         </implementation>
         <implementation render="OGSRenderer" language="GLSL" lang_version="3.0">
             <function_name val="Float4ToFloatW"/>
             <source>
                 <![CDATA[
-// Check whether we're in a GLSL context. If so, we don't expect "floatN"
-// types to exist, so we use "vecN" types instead.
-#if defined(__VERSION__) && (__VERSION__ >= 110)
-#define float4 vec4
-#endif
-
-float
-Float4ToFloatW(float4 input)
+float Float4ToFloatW(vec4 input_is_glsl_kw)
 {
-    return input.w;
+    return input_is_glsl_kw.w;
 }
-
 ]]>
             </source>
         </implementation>
         <implementation render="OGSRenderer" language="HLSL" lang_version="11.0">
             <function_name val="Float4ToFloatW"/>
             <source>
                 <![CDATA[
-// Check whether we're in a GLSL context. If so, we don't expect "floatN"
-// types to exist, so we use "vecN" types instead.
-#if defined(__VERSION__) && (__VERSION__ >= 110)
-#define float4 vec4
-#endif
-
-float
-Float4ToFloatW(float4 input)
-{
-    return input.w;
-}
-
-]]>
-            </source>
-        </implementation>
-        <implementation render="OGSRenderer" language="HLSL" lang_version="10.0">
-            <function_name val="Float4ToFloatW"/>
-            <source>
-                <![CDATA[
-// Check whether we're in a GLSL context. If so, we don't expect "floatN"
-// types to exist, so we use "vecN" types instead.
-#if defined(__VERSION__) && (__VERSION__ >= 110)
-#define float4 vec4
-#endif
-
-float
-Float4ToFloatW(float4 input)
```

Comment:
This PR seems to describe solely refactoring variable name `input` but it also seems to remove quite some lines of code in multiple files. Is that related to the fix? If so, how? And if not, what is the reason for removing it?